examples/prelude/setup.sh

```diff
@@ -1,23 +1,27 @@
-#!/bin/bash
 # Copyright (c) Meta Platforms, Inc. and affiliates.
 #
 # This source code is licensed under both the MIT license found in the
 # LICENSE-MIT file in the root directory of this source tree and the Apache
 # License, Version 2.0 found in the LICENSE-APACHE file in the root directory
 # of this source tree.
 
-set -euo pipefail
+# The commands in this script are to be executed in the current shell
+# and so invoke it via the builtin 'source' command e.g. `source
+# ./setup.sh`. It assumes an opam installation. In addition to
```

Comment:
Normally the `./` bit is only to run things - just `source setup.sh` should be sufficient